package.json

```diff
@@ -54,13 +55,12 @@
     "ts-pattern": "^5.0.6",
     "uuid": "^9.0.1",
     "vaul": "^0.8.0",
-    "zod": "^3.22.4",
-    "prisma": "^5.7.0"
+    "zod": "^3.22.4"
   },
   "devDependencies": {
     "@total-typescript/ts-reset": "^0.5.1",
     "@types/content-disposition": "^0.5.8",
-    "@types/node": "^20",
+    "@types/node": "^20.11.20",
```

Comment:
This file (and presumably `package-lock.json`) is virtually unchanged except for sorting the packages and bumping the `@types/node` version. Is the version bump necessary? If not, I recommend removing both `package.json` and `package-lock.json` from this PR.